lib/v2/bangumionline/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'bangumi.online': {
+        _name: 'アニメ新番組',
+        '.': [
+            {
+                title: 'bangumi online',
```

Comment:
```suggestion
                title: '當季新番',
```
This should match the H3 heading of the docs

Reply:
Changed